Firestore/Source/API/FSTFirestoreComponent.mm

```diff
@@ -21,7 +21,13 @@
 #include <utility>
 
 #import "FirebaseAppCheck/Interop/FIRAppCheckInterop.h"
-#import "FirebaseCore/Internal/FirebaseCoreInternal.h"
+#import "FirebaseCore/Internal/FIRAppInternal.h"
+#import "FirebaseCore/Internal/FIRComponent.h"
+#import "FirebaseCore/Internal/FIRComponentContainer.h"
+#import "FirebaseCore/Internal/FIRComponentType.h"
+#import "FirebaseCore/Internal/FIRDependency.h"
+#import "FirebaseCore/Internal/FIRLibrary.h"
+#import "FirebaseCore/Internal/FIROptionsInternal.h"
```

Comment:
Does it make sense to still have some sort of umbrella header for internal APIs?

Reply:
I'm not sure it's worth it only for Firestore?  The future is modules and Swift